qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+Feature: 1. Sign in to Quantori Scoring Portal with Google account feature
+
+ Background:
+ 	Given Quantori Scoring Portal page (QSPP) exists
+ 	And QSPP is linked to the Internet address https://qa.ddso-spot.quantori.com/
+ 	And Two versions of QSPP could be displayed by the same address: QSPP 'User NOT signed in' and QSPP 'User signed in'
+ 	And On the QSPP 'User NOT signed in' version following components are displayed:
+ 		| Component										|
+ 		| 'Sign in with Google' button					|
+ 		| Quantori Scoring	logotype image				|
+ 	And On the QSPP 'User signed in' version following components are displayed:
```

Comment:
Reduntant info for this feature testing

Reply:
For me it seems important to highlight, what exactly should user see after some actions. I propose we invite Olga to help us resolve this. @shepelevaolga would you, please, help us with this?